.github/workflows/main.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [18.x]
+        node-version: [22.x]
```

Comment:
Nit: unrelated but do we need this matrix here? Think we could drop the matrix all together since this is just for linting?

Reply:
Done